
uint8[] motorval
float32 angle
float32 translation